crates/app/src/metadata.rs

```diff
@@ -23,10 +23,7 @@ impl<T> MetadataKey<T> {
 
 impl<T> Clone for MetadataKey<T> {
     fn clone(&self) -> Self {
-        Self {
-            key: self.key,
-            _phantom: PhantomData,
-        }
+        *self
```

Comment:
Hmm, I don't much like this lint.  I see why it's correct but it's less clear what it's doing and why - if `key` ever changed to a non-`Copy` type, the maintainer would need to reconstruct the clone logic.  I suppose that is too trivial to worry about though.

My preference would be to exclude this lint and keep the code explicit.  But if other folks prefer the elided version then it's no biggie..